setup/prul.ml

```diff
@@ -0,0 +1,115 @@
+let get_some = function
```

Comment:
also prul isn't really the greatest name ...
and didn't you want to keep setup.ml as a single file for easy repl fiddling?